jest.config.js

```diff
@@ -6,10 +6,15 @@ module.exports = {
   rootDir: projectDir,
   moduleNameMapper: {
     "@/(.*)$": path.join(projectDir, "src/$1"),
-    components: "<rootDir>/components",
+    components: "<rootDir>/src/components",
```

Comment:
Talk to Ndifreke and ask what he is doing here so you can refactor this properly